### DISCUSSION TITLE
Many paired Dexcom devices

### DISCUSSION THREAD

**Discussion by andyrozman** [NightscoutFoundation/xDrip#2887] [Q&A] [answered]
Hi !
So today I have noticed that I have 3 paired Dexcom devices on my BT list: DexcomUM, DexcomUX and DexcomG4. This is relatively new phone, so I had one transmiter change, but I did restore phone from old backup (of phone this phone was replacing)...

Does that mean that old transmiters need to be removed manually?

Andy

**Comment by Navid200** [comment, marked as answer]
I don't believe transferring an xDrip database would cause the paired devices to also be paired on the new phone.
You say you have had only one transmitter change on the new phone.  In that case, I would expect to see only two Dexcom** on the list.  If you see 3, are you sure you have not used 3 transmitters?  Even if you temporarily paired an old transmitter, it would still be on the list.

The list will not be automatically cleaned.  So, yes, you need to remove them manually.
